src/components/pages/Home/HomeContainer.js

```diff
@@ -30,12 +41,36 @@ function HomeContainer({ LoadingComponent }) {
 
   return (
     <>
-      {authState.isAuthenticated && !userInfo && (
-        <LoadingComponent message="Fetching user profile..." />
-      )}
-      {authState.isAuthenticated && userInfo && (
-        <RenderHomePage userInfo={userInfo} authService={authService} />
-      )}
+      <Layout>
+        <Layout style={{ height: '100vh' }}>
+          <Content
+            type="flex"
+            style={{ background: 'white', margin: '15%', height: '2.5rem' }}
+          >
+            <Search
+              placeholder="Search for a city"
+              enterButton="Search"
+              size="large"
+              onSearch={value => console.log(value)}
+            />
+            {/* <Button type="primary">Find On Map</Button> */}
+          </Content>
+          <Sider style={{ background: '#C3CFD9' }}></Sider>
+        </Layout>
+        <Footer
```

Comment:
Maybe make footer into a separate component so it can be reused on all pages. Other than that, great work